packages/calcite-components/src/components/action-bar/action-bar.tsx

```diff
@@ -342,8 +342,32 @@ export class ActionBar
       expanded,
       overflowCount,
     });
+
+    this.removeOverflowGroups(height, actionGroups, layout);
   }, overflowActionsDebounceInMs);
 
+  private removeOverflowGroups(
+    height: number,
+    actionGroups: HTMLCalciteActionGroupElement[],
+    layout: "horizontal" | "vertical",
+  ) {
+    let groupHeight = 0;
+    if (layout === "vertical") {
+      actionGroups.forEach((group) => {
+        groupHeight += group.clientHeight;
+      });
+      while (height > groupHeight || Math.abs(height - groupHeight) <= 20) {
+        const group = actionGroups.shift();
+        group.style.display = "none";
+        actionGroups.forEach((group) => {
+          groupHeight += group.clientHeight;
+        });
+      }
+    } else {
+      actionGroups.forEach((a) => (a.style.display = "inline-block"));
```

Comment:
question: the default display for action group is flex. Why is it being set to inline block here?